docs/build.rst

```diff
@@ -103,40 +102,55 @@ the UnifyFS dependencies can then be installed.
 .. code-block:: Bash
 
     $ spack install gotcha
-    $ spack install margo ^mercury+bmi
+    $ spack install mochi-margo ^libfabric fabrics=rxm,sockets,tcp
+    $ spack install spath~mpi
 
 .. tip::
 
-    You can use ``spack install --only=dependencies unifyfs`` to install all
+    You can run ``spack install --only=dependencies unifyfs`` to install all
     UnifyFS dependencies without installing UnifyFS.
 
     Keep in mind this will also install all the build dependencies and
     dependencies of dependencies if you haven't already installed them through
     Spack or told Spack where they are locally installed on your system via a
-    `packages.yaml <https://spack.readthedocs.io/en/latest/build_settings.html#external-packages>`_.
+    packages.yaml_.
 
 Build UnifyFS
 *************
 
-Download the latest UnifyFS release from the `Releases
-<https://github.com/LLNL/UnifyFS/releases>`_ page or clone the develop branch
-from the UnifyFS repository `https://github.com/LLNL/UnifyFS <https://github.com/LLNL/UnifyFS>`_.
+Download the latest UnifyFS release from the Releases_ page or clone the develop
+branch from the UnifyFS repository
+`https://github.com/LLNL/UnifyFS <https://github.com/LLNL/UnifyFS>`_.
 
 Load the dependencies into your environment and then
 configure and build UnifyFS from its source code directory.
 
 .. code-block:: Bash
 
     $ spack load gotcha
-    $ spack load mercury
     $ spack load argobots
+    $ spack load mercury
     $ spack load margo
+    $ spack load spath
     $
     $ ./autogen.sh
-    $ ./configure --prefix=/path/to/install
+    $ ./configure --prefix=/path/to/install --with-gotcha=${gotcha_install} --with-spath=${spath_install}
     $ make
     $ make install
 
+Alternatively, UnifyFS can be configured using ``CPPFLAGS`` and ``LDFLAGS``:
+
+.. code-block:: Bash
+
+    $ ./configure --prefix=/path/to/install CPPFLAGS="-I${gotcha_install}include -I{spath_install}/include" LDFLAGS="-L${gotcha_install}/lib64 -L${spath_install}/lib64
```

Comment:
@CamStan you're missing a `/` in gotcha include

Reply:
Yes, yes I am.
Added it.